.github/renovate.json5

```diff
@@ -298,17 +316,18 @@
       ],
       "allowedVersions": "<1.21",
       "matchBaseBranches": [
-        "v1.13"
+        "v1.14",
+        "v1.13",
+        "v1.12"
```

Comment:
1.14, 1.13, and 1.12 were all bound to Go 1.20.x but through separate renovate statements previously. I've combined those three cases together here, so that whenever Go 1.20 goes out of support we can bump multiple at once.